canonical/snapcraft.io
# Add store side navigation

## Done
Added the store selector and section nav to the sidebar

## How to QA
- Go to https://snapcraft-io-4546.demos.haus/admin
- Check that the stores list is in a select field and that changing the value changes the store
- Check that the "Members" and "Settings" links are in the sidebar for the stores you have permissions to see them in
- **Note**: Logout and account don't work as yet - this is known

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-8960
Fixes https://warthogs.atlassian.net/browse/WD-8959
